includes/html/api_functions.inc.php

```diff
@@ -1737,6 +1737,44 @@ function list_oxidized(Illuminate\Http\Request $request)
     return response()->json($return, 200, [], JSON_PRETTY_PRINT);
 }
 
+function get_oxidized_state(Illuminate\Http\Request $request): JsonResponse
+{
+    $hostname = $request->route('hostname');
+    $device_id = ctype_digit($hostname) ? (int)$hostname : getidbyname($hostname);
+    $oxidized_disabled = DeviceCache::get($device_id)->getAttrib('override_Oxidized_disable');
+    $result = match ($oxidized_disabled) {
+        'true' => true, // Oxidized is disabled if the override is True
+        'false' => false, // Oxidized is not disabled if the override is False
+        default => false  // Attrib is not set, so Oxidized can't be disabled
+    };
+    return api_success($result, 'oxidized_disabled');
+}
+
+function set_oxidized_state(Illuminate\Http\Request $request): JsonResponse
+{
+    $hostname = $request->route('hostname');
+    $device_id = ctype_digit($hostname) ? (int)$hostname : getidbyname($hostname);
+    $data = json_decode($request->getContent(), true);
+    $content = $data['oxidized_disabled'];
```

Comment:
You could get an array key doesn't exist here.

If this is a simple boolean toggle, why is it nested in an array?